crates/sui-core/src/authority_aggregator.rs

```diff
@@ -1398,11 +1397,6 @@ where
             Ok(PlainTransactionInfoResponse::Signed(signed)) => {
                 debug!(?tx_digest, name=?name.concise(), weight, "Received signed transaction from validator handle_transaction");
                 self.handle_transaction_response_with_signed(state, signed)
-                    .tap_ok(|opt_cert| {
-                        if let Some(cert) = opt_cert.as_ref() {
-                            debug!(?tx_digest, ?cert, "Collected tx certificate for digest")
```

Comment:
Do we have other debug logs that indicate when the cert is collected? Just want to make sure we still have a log line that we can tell the cert is ready

Reply:
yes https://github.com/MystenLabs/sui/blob/main/crates/sui-core/src/authority_aggregator.rs#L1382